front/lib/api/assistant/pubsub.ts

```diff
@@ -230,6 +230,19 @@ export async function retryAgentMessageWithPubSub(
                 resolve(new Ok(event.message));
                 break;
               }
+              case "agent_message_error": {
```

Comment:
What will happen client side in the handler of the retry if we get that error?

Reply:
For the retry case, the client side does not do anything with the error. It relies on the streaming to process the error.

For the edit case, the client side shows an error, but double submitting is now blocked client side.

Reply:
Ok so we kind of have a assumption here that we should write down. We can't throw these errors before we emit another event.

This leads me to think that we should put the try/catch around the sequelize and not the whole function no?

Reply:
> Ok so we kind of have a assumption here that we should write down. We can't throw these errors before we emit another event.

I don't understand what you mean here.
Which assumption should we write down?
And what do you mean by we can't throw these errors before we emit another event?